### PR TITLE
sys/usbus: Expose configurations to Kconfig

### PR DESCRIPTION
**Pull request by leandrolanzieri**
### Contribution description
This moves `USBUS_AUTO_ATTACH` and `USBUS_EP0_SIZE` configuration options to the `CONFIG_` namespace, and exposes them to Kconfig.

### Testing procedure
Test with `examples/usbus_minimal`: Should work normally and you should be able to change options via Kconfig.

### Issues/PRs references
Part of #12888

### CODE REVIEW

**Comment by tcschmidt** [comment]
@cgundogan @bergzand what about this: can we push for the release?

File: sys/usb/usbus/Kconfig [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    bool "Auto attach"
+    default y
+    help
+        When set, the USBUS thread will automatically enable the USB pull-up
+        resistor after initializing the thread. This will signal to the host
+        that the USB peripheral is ready for use.
+
+config USBUS_EP0_SIZE
+    int "Endpoint 0 buffer size"
+    default 64
```
Comment (bergzand) [inline sys/usb/usbus/Kconfig:25]: As per USB spec, the only valid values for this buffer size are 8, 16, 32 and 64. Should this option be a choice type with the valid buffer options?

Comment (leandrolanzieri) [reply sys/usb/usbus/Kconfig]: Yes, sure. I was not aware of that constraint. I modeled this as a choice now.

**Comment by bergzand** [comment]
The config options look good to me. I've played around with the configuration options and things look fine. I've asked @cgundogan to give this a final test.

**Comment by leandrolanzieri** [comment]
Thanks everyone for reviewing and testing!

**Pull request merged**